src/NuGet.Core/NuGet.Commands/RestoreCommand/DependencyGraphResolver.cs

```diff
@@ -1520,23 +1439,25 @@ private class DependencyGraphItem
             public HashSet<LibraryDependencyIndex>? Suppressions { get; set; }
         }
 
-        private class FindLibraryEntryResult
+        private class ResolvedLibraryEntryResult
         {
             private LibraryDependencyIndex[] _dependencyIndices;
             private LibraryRangeIndex[] _rangeIndices;
 
-            public FindLibraryEntryResult(
-                LibraryDependency libraryDependency,
-                GraphItem<RemoteResolveResult> resolvedItem,
-                LibraryDependencyIndex itemDependencyIndex,
-                LibraryRangeIndex itemRangeIndex,
+            public ResolvedLibraryEntryResult(
+                GraphNode<RemoteResolveResult> node,
```

Comment:
This type change had me thinking. 
A graph node is an item that has the parents/children within a graph.
It seems like something that wouldn't be cached normally. 
Then later I looked and it seems like we *only* ever use the item and not the node, but I'm not 100% sure.

Reply:
I think we should clean that up in a future refactoring

Reply:
Can you please make sure it's tracked somewhere.